modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableMessageGroup.java

```diff
@@ -125,4 +125,23 @@ public interface TableMessageGroup {
      * Message type for {@link SnapshotTxDataResponse}.
      */
     short SNAPSHOT_TX_DATA_RESPONSE = 16;
+
+    /**
+     *
+     */
+    public interface Commands {
```

Comment:
Please add a little comment to the class and for each constant. `Mesage type for {@link Foo}.` is enough, like for other messages